Makefile

```diff
@@ -87,21 +87,31 @@ run: generate fmt vet manifests
 # Install CRDs into a cluster
 install: manifests
 	kustomize build config/crd | kubectl apply -f -
-	# Remove the following when https://github.com/projectcontour/contour-operator/issues/42 merges.
-	kubectl apply -f https://raw.githubusercontent.com/projectcontour/contour/main/examples/contour/01-crds.yaml
 
 # Uninstall CRDs from a cluster
 uninstall: manifests
 	kustomize build config/crd | kubectl delete -f -
 
-# Deploy controller in the configured Kubernetes cluster in ~/.kube/config
+# Generate the example operator manifest
+example:
+	cd config/manager && kustomize edit set image controller=${IMAGE}:${VERSION}
```

Comment:
Is the idea that you'd use this locally when developing to change the version / image? I ask because the other release tooling will be used when a tag is created.

Reply:
@stevesloka thanks for catching this. I removed `kustomize edit set image controller=${IMAGE}:${VERSION}` from the different make targets. Having kustomze set the image/version should no longer be needed when https://github.com/projectcontour/contour-operator/pull/116 merges. Is the Envoy image tag manually managed in Contour?